pkg/login/auth.go

```diff
@@ -64,3 +82,20 @@ func loginUsingGrafanaDB(query *LoginUserQuery) error {
 	query.User = user
 	return nil
 }
+
+func validateLoginAttemptCount(username string) error {
+	loginAttemptCountQuery := m.GetUserLoginAttemptCountQuery{
+		Username: username,
+		Since:    time.Now().Add(time.Minute * -5),
```

Comment:
Should it be possible to configure this as well?

Reply:
No, not in the initial implementation